test/benchmark/common/utils.go

```diff
@@ -18,3 +23,49 @@ func CreateTSDB(v3ioConfig *config.V3ioConfig, newTsdbPath string) error {
 func DeleteTSDB(adapter *tsdb.V3ioAdapter, deleteConf bool, force bool) {
 	adapter.DeleteDB(deleteConf, force)
 }
+
+func ValidateCountOfSamples(adapter *tsdb.V3ioAdapter, expected int, startTimeMs, endTimeMs int64) error {
```

Comment:
Maybe pass interface TB (mutual between T and B tests) so that the valudation will fail the test directly instead of checking it on every test

Reply:
in that case you won't be able to use this validation outside of the test (as an API)
if this function is going to be used for tests only we can do it